.github/workflows/tests.yml

```
@@ -0,0 +1,36 @@
+name: CI/CD
```

This is just targeting the integration so should probably be just CI
```suggestion
name: CI
```